Toolkit/ArcGISToolkit.xcodeproj/project.pbxproj

```diff
@@ -507,12 +519,16 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
-					"$(USER_LIBRARY_DIR)/SDKs/ArcGIS/iOS/Frameworks/Dynamic",
+					"$(USER_LIBRARY_DIR)/SDKs/ArcGIS/Frameworks",
 				);
```

Comment:
Setting a search path for the framework is no longer necessary. Please select the setting and press Delete to revert to using the default value.

![Screen Recording 2020-11-11 at 12 50 27 PM 2020-11-11 12_50_57](https://user-images.githubusercontent.com/2257493/98863047-a6bcb000-241c-11eb-8555-487f55753272.gif)o

Same with the examples app.